bindings/python/iota_sdk/client/client.py

```diff
@@ -103,11 +104,17 @@ def get_remaining_nano_seconds(duration: timedelta):
             client_config['remote_pow_timeout'] = {'secs': int(client_config['remote_pow_timeout'].total_seconds(
             )), 'nanos': get_remaining_nano_seconds(client_config['remote_pow_timeout'])}
 
+        # Delete client_handle, because it's not needed here and can't be serialized
+        if "client_handle" in client_config:
+            del client_config["client_handle"]
```

Comment:
Btw would it makes sense to move all deletes to the top? So they are all in one place?